cms/djangoapps/contentstore/views/course.py

```diff
@@ -927,6 +982,7 @@ def advanced_settings_handler(request, course_key_string):
                 try:
                     # Whether or not to filter the tabs key out of the settings metadata
                     filter_tabs = _config_course_advanced_components(request, course_module)
+                    filter_tabs = _config_course_settings(request, course_module, filter_tabs)
```

Comment:
this has been reworked in https://github.com/edx/edx-platform/commit/c384ee50b77e52fdf5cf78b7e3caa5f04fe5eef2